### ISSUE TITLE
Proxy authentication not sended

### ISSUE THREAD

**Issue by rricordeau** [usebruno/bruno#3217]
### I have checked the following:

- [X] I use the newest version of bruno.
- [X] I've searched existing issues and found nothing related to my issue.

### Describe the bug

I'm using bruno to call an api over a squid proxy with basic auth. 

when I use bruno I had an error that login ar not sended.

### Screenshots/Live demo link

![image](https://github.com/user-attachments/assets/43a7c1be-d7f7-4647-8605-4be2c277be85)

**Comment by helloanoop** [comment]
@rricordeau Thank you for reporting this issue.

Could you provide some additional details to help us investigate further?

1. Was this functionality previously working in Bruno and has now started occurring in a newer version?
2. Are you configuring the proxy settings manually (refer to the screenshot below), or are you relying on the system-level proxy settings?

<img width="713" alt="image" src="https://github.com/user-attachments/assets/3ce11d6e-2375-4657-963b-e669969f3dcb">

**Comment by rricordeau** [comment]
It never worked on my computer

I configure with the interface in your screenshot
![image](https://github.com/user-attachments/assets/889b2afd-f201-48a6-8844-1b2f80c5b736)


I'm using version 1.33.0 Golden Edition

**Comment by helloanoop** [comment]
@rricordeau, are you able to successfully hit the API endpoint using curl? If so, I would recommend trying the System Proxy Mode in Bruno.

Also, is this an issue you're encountering specifically in Bruno, or has it worked fine in other API clients you've used? Just trying to narrow down the cause here.

**Comment by rricordeau** [comment]
Yes, in curl, I set the proxy with http://login:password@myproxy:3128 and it work without any trouble

In bruno, I can use a proxy without proxy auth without any problem.
